src/Form/Type/ModelAutocompleteType.php

```diff
@@ -156,6 +158,7 @@ public function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('model_manager', ModelManagerInterface::class);
         $resolver->setAllowedTypes('class', 'string');
         $resolver->setAllowedTypes('property', ['string', 'array']);
+        $resolver->setDeprecated('quiet_millis', 'Using `quiet_millis` is deprecated! Use `delay` instead!'); // NEXT_MAJOR: Remove this deprecation notice.
```

Comment:
The signature of the method `setDeprecated` changed between symfony 4 and symfony 5/6.

Can you do the same than I did in https://github.com/sonata-project/SonataBlockBundle/pull/997 ?

Reply:
Gotcha! Should I input any specific version like you did there?

Reply:
Use `4.x`, I'll change the 4.x into the right version when the release will be made